src/html/index.html

```diff
@@ -0,0 +1,71 @@
+{#
+  Nunjucks Templating!
+  Homepage: https://mozilla.github.io/nunjucks/
+  Docs: https://mozilla.github.io/nunjucks/api.html
+#}
+
+{% extends 'layouts/application.html' %}
+
+{% block content %}
+<div id="top-fold">
+  <div id="title-box">
+  <p id="subtitle">A Viget Exploration</p>
+    <h1 id="title">
+      <a href="/">
+        <img src="images/title.svg" alt="Improving Medical Adherence" />
+      </a>
+    </h1>
+    <img src="images/yellow-pill.svg" alt="Yellow pill." id="yellow-pill">
+    <img src="images/pills.svg" alt="Green and light yellow pills." id="pills">
+    <img src="images/more-pills.svg" alt="Three pill tablets." id="more-pills">
+    <img src="images/purple-pills.svg" alt="Two purple pills." id="purple-pills">
+  </div>
+    <nav class="pillbar">
+      <ul>
+        <a href="#box-one"><li class="left-end active">The roblem</li></a>
+        <a href="#key-factors" onclick="scroll(1)"><li>Key Factors</li></a>
+        <a href="#questions"><li>The Question</li></a>
+        <a href="#solution"><li class="right-end ">The Solution</li></a>
+      </ul>
+    </nav>
```

Comment:
a little unusual, but I'd consider using two lists here, the first with the red bg and the first three items, the second with the dark bg and the last time. that way there's no chance of text crossing that dividing line